distributed/diagnostics/tests/test_packageinstall.py

```diff
@@ -0,0 +1,245 @@
+from __future__ import annotations
```

Comment:
In the future could we have the verbatim move of unit test from one file to another and any changes to them in two separate PRs? This makes it quite difficult to review.

Reply:
Sorry about that!